test/Makefile

```
@@ -29,12 +29,12 @@ build:
 	$(GINKGO) build -tags=integration_tests
 	$(QUIET)$(MAKE) -C bpf/
 
-test: run k8s
+test: run k8sT
 
 run:
 	KERNEL=net-next ginkgo --focus "Runtime" --tags integration_tests -v -- --cilium.provision=$(PROVISION) --cilium.registryCredentials=$(REGISTRY_CREDENTIALS)
 
-k8s:
+k8sT:
```

```suggestion
k8s-test:
```

is probably a better name for the target so it's more discoverable.